src/opentimelineview/timeline_widget.py

```diff
@@ -498,7 +498,14 @@ def mouseReleaseEvent(self, mouse_event):
         self.setDragMode(QtWidgets.QGraphicsView.NoDrag)
 
     def wheelEvent(self, event):
-        scale_by = 1.0 + float(event.delta()) / 1000
+        try:
+            delta_point = event.angleDelta()
+            delta = delta_point.y()
+
+        except AttributeError:
+            delta = event.delta()
```

Comment:
Could you put a comment explaining which of these is for PySide 2 vs 6?

Reply:
Added